druid-handler/src/test/org/apache/hadoop/hive/druid/serde/TestDruidSerDe.java

```diff
@@ -86,7 +85,7 @@
 import org.apache.druid.query.topn.TopNResultValue;
 import org.junit.rules.ExpectedException;
 
-/**
+/**Z
```

Comment:
can be reverted

Reply:
thanks for catching this, reverted in [this](https://github.com/apache/hive/pull/5404/commits/85943a9c46335277d408d711421e507d8b3a7308)